bin/scoop.ps1

```diff
@@ -33,22 +32,22 @@ switch ($SubCommand) {
 
         Get-LocalBucket | ForEach-Object {
             $bucketLoc = Find-BucketDirectory $_ -Root
-            if ((Test-Path (Join-Path $bucketLoc '.git')) -and (Test-CommandAvailable git)) {
+            if ((Test-Path "$bucketLoc\.git") -and (Test-CommandAvailable git)) {
                 Write-Host "'$_' bucket:"
                 Invoke-Expression "git -C '$bucketLoc' --no-pager log --oneline HEAD -n 1"
                 Write-Host ''
             }
         }
     }
-    ({ $SubCommand -in (commands) }) {
-        if ($arguments.Count -gt 0 -and $arguments[0] -in @('-h', '--help', '/?')) {
-            exec 'help' @($SubCommand)
+    ({ $subCommand -in (commands) }) {
+        if ($arguments -in @('-h', '--help', '/?')) {
```

Comment:
@niheaven If you make this change, it goes wrong on a call like `scoop install -h --invalid-flag`:

```
scoop install: Option -h not recognized.
```

For best practice in CLI, if a "help" flag is specified, we should ignore any invalid flags by default. The original change I made is exactly for this use case.

Reply:
We can change the line as `if ($arguments[0] -in @('-h', '--help', '/?')) {` if we have changed [this](https://github.com/ScoopInstaller/Scoop/pull/4931/files#r874819487) as well.

Reply:
See recent commit [127f231](https://github.com/ScoopInstaller/Scoop/pull/4931/commits/127f231a792c919ac807bfbaecf105ebcf3e34f2)

Reply:
Validated again and updated the change log.